src/pytorch_lightning/serve/servable_module.py

```diff
@@ -0,0 +1,90 @@
+from typing import Any, Callable, Dict, Tuple
+
+import torch
+
+
+class ServableModule(torch.nn.Module):
+
+    """The ServableModule provides a simple API to make your model servable.
+
+    .. warning::
+
+        This is currently an experimental feature and API changes are to be expected.
+
+    Here is an example on how to use the `ServableModule` module.
+
+    .. code-block:: python
+
+        from typing import Dict, Any, Callable
+
+        import torch
+
+        from pytorch_lightning import seed_everything, Trainer
+        from pytorch_lightning.demos.boring_classes import BoringModel
+        from pytorch_lightning.serve.servable_module_validator import ServableModule, ServableModuleValidator
+
+
+        class ServableBoringModel(BoringModel, ServableModule):
+            def configure_payload(self) -> Dict[str, Any]:
+                return {"body": {"x": list(range(32))}}
+
+            def configure_serialization(self) -> Tuple[Dict[str, Callable], Dict[str, Callable]]:
+                def deserialize(x):
+                    return torch.tensor(x, dtype=torch.float)
+
+                def serialize(x):
+                    return x.tolist()
+
+                return {"x": deserialize}, {"output": serialize}
+
+            def serve_step(self, x: torch.Tensor) -> Dict[str, torch.Tensor]:
```

Comment:
I think the example should also use `configure_response` to show that a user doesn't have to do the output validation.

Reply:
Done !